projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.component.ts

```diff
@@ -253,7 +253,7 @@ export class IgxExpansionPanelComponent implements IgxExpansionPanelBase, AfterC
         this.cdr.detectChanges();
         this.playOpenAnimation(
             () => {
-                this.onExpanded.emit({ event: evt, panel: this });
+                this.onExpanded.emit({ event: evt, panel: this, owner: this });
```

Comment:
please update the comments for this event, as  it is not pointing in TS part the correct type of the event - `IExpansionPanelEventArgs`. It looks like this now:
```
     * ```typescript
     *  handleExpanded(event: {
     *  panel: IgxExpansionPanelComponent,
     *  event: Event
     * })
     * ```
```